Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -818,6 +820,13 @@ public class Event: NosManagedObject, VerifiableEvent {
     @MainActor var loadingViewData = false
     @MainActor var attributedContent = LoadingContent<AttributedString>.loading
     @MainActor var contentLinks = [URL]()
+    @MainActor private(set) var quotedNoteID: RawEventID? {
+        didSet {
+            if quotedNoteID != nil {
+                Task { await loadFirstQuotedNote() }
```

Comment:
Given that loadFirstQuotedNote() makes requests to relays, can you refactor this so setting quotedNoteID doesn't have side effects like this? It makes it harder to track what's happening in the app. I'm fine if that means having Task { await loadFirstQuotedNote() } in a couple of places. I think it would make the code easier to understand.

Reply:
Moved it just to one spot.